doc/source/r_interface.rst

```diff
@@ -60,7 +60,7 @@ can be replaced with::
 
     from rpy2.robjects import r
     r.data('iris')
```

Comment:
Yep, the docs were still 'to do'. What I was thinking to do is update the current examples to just use the corresponding `rpy2` functions. So then there is still some documentation on how to use it (for the updating, I can refer to docs of previous version).